lib/topological_inventory/providers/common/operations/source.rb

```diff
@@ -17,13 +17,17 @@ class Source
         }.freeze
 
         LAST_CHECKED_AT_THRESHOLD = 5.minutes.freeze
+        AUTH_NOT_NECESSARY = "n/a".freeze
 
-        attr_accessor :params, :request_context, :source_id
+        attr_accessor :params, :request_context, :source_id, :account_number, :receptor_client
 
-        def initialize(params = {}, request_context = nil)
+        def initialize(params = {}, request_context = nil, receptor_client = nil)
```

Comment:
this change is not needed now (regarding https://github.com/RedHatInsights/topological_inventory-ansible_tower/pull/107)

Reply:
Updated!